server/lib/benches/image_benches.rs

```diff
@@ -1,134 +1,135 @@
-/// This file contains benchmarks for the image module so we can work out the best order to run things in
-use std::time::Duration;
+// /// This file contains benchmarks for the image module so we can work out the best order to run things in
 
-use criterion::{black_box, criterion_group, criterion_main, Criterion};
-use kanidmd_lib::valueset::image::jpg;
-use kanidmd_lib::valueset::image::png;
+// use std::time::Duration;
```

Comment:
Did we want to delete instead of commenting out?

Reply:
Not even a 1% gain in lines reduced :P